sample_apps/run_example.sh

```diff
@@ -809,13 +812,27 @@ function setup_cca_emulated_islet_shim_bins() {
 }
 
 # ###########################################################################
-function embed_policy_in_example_app() {
+function gen_policy_key_for_example_app() {
     run_cmd
     run_pushd "${PROV_DIR}"
 
+    python_flag=""
+    init_cert_flag=""
+    outfile="policy_key.cc"
+
+    # Invoke utility with python-specific arguments.
+    # Use upper-case names to avoid pylint errors.
+    if [ "${SampleAppName}" = "simple_app_python" ]; then
+        python_flag="--python"
```

Comment:
The `embed_policy_key.exe` has been enhanced to take `--python`-specific flags which changes the format in which output is rendered.